suite-common/wallet-utils/src/validation.ts

```diff
@@ -4,7 +4,7 @@ import { Account } from '@suite-common/wallet-types';
 
 const getNetworkType = (symbol: Account['symbol']) => {
     if (symbol === 'regtest') return symbol;
-    const testnets = ['test', 'txrp', 'tgor', 'tada'];
+    const testnets = ['test', 'txrp', 'tsep', 'tgor', 'tada'];
```

Comment:
Also possible to dynamically pull all testnets from the config.

Reply:
d159ece4d3309f70414e05e094d148f670899429